lib/task/import/validate/csvBaseValidator.class.php

```diff
@@ -91,9 +94,45 @@ public function appendDuplicatedColumnError(string $columnName)
         $this->testData->addResult(sprintf('Unable to validate because of duplicated columns in CSV.'));
     }
 
+    public function setHeader($header)
```

Comment:
Nitpick: It would be nice to specify the type of `$header` parameter.  I'm guessing it's an array?

Reply:
Good idea - fixed